src/ashley/templatetags/custom_tags.py

```diff
@@ -0,0 +1,39 @@
+"""Ashley template tags to display forum content"""
+from django import template
+from machina.core.db.models import get_model
+
+from ashley.defaults import _FORUM_ROLE_INSTRUCTOR
+
+LTIContext = get_model("ashley", "LTIContext")
+Topic = get_model("forum_conversation", "Topic")
+
+register = template.Library()
+
+
+@register.filter()
+def is_user_instructor(topic_pk, user):
+    """
+    This will return a boolean indicating if the passed user is instructor
+    for the given topic.
+    Usage::
+        {% if topic_pk|is_user_instructor:user %}...{% endif %}
+    """
+    # get the list of lti concerns by this forum, one forum can have multiple LTI Context
+    # if the user is instructor in one of them then he is considered
```

Comment:
I'm confused: do we still want this now that @madmatah's PR https://github.com/openfun/ashley/pull/143 is limiting a user to a context for a given LTI connection?

Reply:
This aspect will disappear but as it is still for now the attended behaviour, I think until it evolved I have to test this.

Reply:
I'm afraid he will merge his before yours ^^

Reply:
I'm not sure regarding your return, I have a good chance !